.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: ci
+
+"on":
+  push:
+    paths:
+      - ".github/workflows/ci.yml"
+      - "lib/**"
+      - "*.gemspec"
+      - "spec/**"
+      - "Rakefile"
+      - "Gemfile"
+      - ".rubocop.yml"
+  pull_request:
+    branches:
+      - master
+  create:
```

Comment:
I've noticed that this `create:` reference causes two CI runs for each push.

Reply:
yeah, I think we shouldn't need that